docs/developer-docs/web-apps/verifiable-credentials/relying-party.mdx

```diff
@@ -0,0 +1,272 @@
+---
+keywords: [intermediate, tutorial, identity, credentials, verifiable credentials, relying party]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Become a relying party
+
+<MarkdownChipRow labels={["Intermediate", "Tutorial"]} />
+
+## Overview
+
+A **relying party** is a service or app that requests a user's verifiable credentials from the [issuer](issuer.mdx) API. Relying parties use the identity provider API to communicate with the issuer API using an **identity provider** such as Internet Identity. Identity providers must support the [verifiable credentials specification](https://github.com/dfinity/internet-identity/blob/main/docs/vc-spec.md). The identity provider API uses the [`window.postMessage()`](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage) interface.
+
+Internet Identity is used as the identity provider in this tutorial.
```

Comment:
Is this a tutorial? I would remove this sentence if we already mention "such as Internet Identity" above.

Reply:
Removed this sentence